front/src/routes/integration/all/mcp/actions.js

```diff
@@ -0,0 +1,80 @@
+import { SESSION_TOKEN_TYPES } from '../../../../../../server/utils/constants';
```

Comment:
We don't use unistore anymore for functions, I recommend to simply use class functions, could you migrate those functions simply to the `class MCPIntegration extends Component {`

(Ask any AI to migrate that, it'll be super easy)